src/main/java/seedu/address/storage/XmlAddressBookStorage.java

```diff
@@ -74,4 +74,9 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, String filePath) th
         XmlFileStorage.saveDataToFile(file, new XmlSerializableAddressBook(addressBook));
     }
 
+    @Override
+    public void backupAddressBook(ReadOnlyAddressBook addressBook) throws IOException {
+        // does nothing yet.
```

Comment:
Perhaps you shouldn't include this override in your PR yet since it's currently not implemented. I think this one of the reasons why coverage is going down.